plugins/web/opentelemetry-instrumentation-user-interaction/test/userInteraction.nozone.test.ts

```diff
@@ -410,6 +415,122 @@ describe('UserInteractionInstrumentation', () => {
       });
     });
 
+    it('should handle interactions listened on document - react < 17', done => {
+      const btn1 = document.createElement('button');
+      btn1.setAttribute('id', 'btn1');
+      document.body.appendChild(btn1);
+      const btn2 = document.createElement('button');
+      btn2.setAttribute('id', 'btn2');
+      document.body.appendChild(btn2);
+
+      const listener = (event: MouseEvent) => {
+        switch (event.target) {
+          case btn1:
+            getData(FILE_URL, () => {
+              sandbox.clock.tick(10);
+            }).then(() => {});
+            break;
+          case btn2:
+            getData(FILE_URL, () => {
+              sandbox.clock.tick(10);
+            }).then(() => {});
+            break;
+        }
+      };
+
+      document.addEventListener('click', listener);
+
+      try {
+        btn1.click();
+        btn2.click();
```

Comment:
I appreciate this test (having two targets and ensuring that the span names properly line up)!